docs/db-migration-guide.ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Charmed Kubeflow Databases Migration Guide\n",
+    "\n",
+    "There are two databases in Charmed Kubeflow - `katib-db` and `kfp-db`. In Charmed Kubeflow 1.7 these MariaDB databases deployed by `charmed-osm-mariadb-k8s` charm. Since release 1.7 there is an option to relate charm to new MySQL database that is deployed by `mysql-k8s` charm. In cases where previous deployment has MariaDB a migration of database is required."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
```

Comment:
I'd add a subsection before or after "Prerequisites" that helps users know under which circumstances they are supposed to migrate, for instance, I'd give instructions on how to check which database they have deployed and based on that they can check whether they need to migrate.

Reply:
I will use Colm's suggestions and just indicate this in the begging of the doc. Basically saying that when upgrading to 1.8 you must migrate databases if they are MariaDB. Will add the the check for type of DB in Prerequisites.